__tests__/topics.test.js

```diff
@@ -210,6 +210,46 @@ describe('/api/articles/:article_id', () => {
   });
 });
 
+
+
+describe('/api/users', () => {
+  describe('GET /api/users', () => {
+    test('200: status code and contain the expected data type and fields', async () => {
+      const res = await request(app).get('/api/users');
+      expect(res.statusCode).toBe(200);
+      expect(Array.isArray(res.body)).toBe(true);
```

Comment:
I would check the length of the array, if it's empty your expects inside the forEach will give you a false positive